parser/src/token.rs

```diff
@@ -999,6 +999,7 @@ mod test {
     }
 
     #[test]
+    #[ignore]
```

Comment:
What is the best way to fix this test?

Reply:
Not sure atm, I will look into it

Reply:
If you cherry-pick https://github.com/Marwes/gluon/commit/fafe7b342486a23f81fcbab9568dd2e489e0be63 it should work.